gamemode/config/config.lua

```diff
@@ -323,6 +323,9 @@ GM.Config.chatCommandPrefix = "/"
 GM.Config.F1MenuHelpPage = "https://darkrp.miraheze.org/wiki/Main_Page"
 GM.Config.F1MenuHelpPageTitle = "DarkRP Wiki"
 
+-- The sound that plays when you get a DarkRP notification
+GM.Config.NotifSound = "buttons/lightswitch2.wav"
```

Comment:
Could you change this to `GM.Config.notificationSound`? The capitalization is a bit of a mess, but I think this would be most consistent with the other options.

Reply:
Done 👍